pkg/services/rendering/interface.go

```diff
@@ -75,14 +82,9 @@ type SanitizeSVGResponse struct {
 	Sanitized []byte
 }
 
-type CSVOpts struct {
-	TimeoutOpts
-	AuthOpts
-	Path            string
-	Encoding        string
-	Timezone        string
-	ConcurrentLimit int
-	Headers         map[string][]string
+type Result struct {
```

Comment:
Generic result to reuse in all cases. I didn't change `RenderResult ` and `RenderCSVResult` to avoid changing all returning types.